meta-mender-core/recipes-mender/mender/mender.inc

```diff
@@ -22,6 +22,7 @@ B = "${WORKDIR}/build"
 inherit go
 
 SRC_URI_append = " file://mender.service \
+                   file://0001-Allow-mulitple-digit-partition-numbers.patch \
```

Comment:
This should be submitted to the mender repository, not as a patch in meta-mender.

Reply:
And again I see you already did that! Never mind me, I'm overeager on the comment button! :-)